features/shop/checkout_with_paypal_payment.feature

```diff
@@ -0,0 +1,28 @@
+@ui-checkout @insulated @mink:firefox
+Feature: Checkout with PayPal Express Checkout
+    In order to buy products
+    As a Customer
+    I want to be able to pay with PayPal Express Checkout
+
+    Background:
+        Given the store is operating on a single channel
```

Comment:
We have a new tool on master now. To format features use `bin/kawaii gherkin:fix --align=left features/`